accepted/0000-hosted-git-dependencies.md

```diff
@@ -0,0 +1,84 @@
+- Start Date: 2017-08-08
+- RFC PR:
+- Yarn Issue:
+
+# Summary
+
+Clarify what is a *hosted* git dependency, which dependency patterns should be considered as *hosted*, and what are the differences with a regular git dependency.
+
+# Motivation
+
+*Hosted* git dependencies are dependencies hosted on Github, Gitlab or Bitbucket that can download the manifest and dependency content with a HTTPS request instead of using the `git` command.
+
+Because they are treated differently, they do not benefit from all the features of a regular git dependency (like yarnpkg/yarn#3553), or introduce some incompatibilities (yarnpkg/yarn#3923).
+Also these differences are poorly documented and hard to identify from the code.
+
+The **main goal** is to ensure that the same result can be expected by installing a *hosted* git dependency or a regular git dependency.
+
+Secondly, exhaustively document which patterns are resolved as *hosted* dependencies, what are the implications for private repositories, and how to write a pattern to be explicitly *regular*.
+Plus it would be nice to synchronise with `npm` (npm/hosted-git-info ?)
+
+At last, as much as possible, let the feature be extensible to other git hosts in the future (for example, self-hosted gitlab or bitbucket).
+
+# Detailed design
+
+As a first step, the existing implementation for *hosted* git dependencies should be removed, and patterns that previously matched a hosted dependency should resolve to a git repository.
```

Comment:
Could you justify why the code for hosted git dependencies should be removed?
Are there specific issues caused by the current implementation?
Can it be just modified?